frontend/src/components/molecules/TokenLink.vue

```diff
@@ -2,12 +2,14 @@
 	<div class="inline-block whitespace-nowrap">
 		<TokenIcon class="h-5 inline align-text-top" />
 		<LinkButton class="numerical px-2" @blur="emitBlur" @click="handleOnClick">
-			<div text-class="text-theme-body">
-				{{ props.tokenAddress }}
-			</div>
+			<Tooltip :text="tokenAddressLowerCase" text-class="text-theme-body">
+				<div class="token-address truncate">
+					{{ tokenAddressLowerCase }}
+				</div>
+			</Tooltip>
```

Comment:
I don't think token address should be lowercase. It is case sensitive (like account address). The token id, which is in base16/hex shoudl be lowercase though.